app/views/pages/report.ejs

```diff
@@ -499,6 +499,7 @@
   </div>
 </section>
 
+<script src="https://cdn.jsdelivr.net/npm/flatpickr"></script>
```

Comment:
Please use https://www.srihash.org/ to include the SRI hash for external resources.

Reply:
Do I wait for someone to host this on https://cdn.coronastatus.eu before I update the src and SRI hash?